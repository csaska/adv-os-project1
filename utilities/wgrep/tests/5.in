version https://git-lfs.github.com/spec/v1
oid sha256:fa08a4ae0fe02bf7b62a021b9e7531bb5a378ead41c323cc8b805c7a900fb60b
size 473
