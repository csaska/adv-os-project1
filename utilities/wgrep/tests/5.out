version https://git-lfs.github.com/spec/v1
oid sha256:08ab3b2851fdc25c285f13d90b617909e538702535e7f5c22129a9f0e4a56506
size 262
